gms/api/src/main/pegasus/com/linkedin/dataset/Dataset.pdl

```diff
@@ -122,4 +123,9 @@ record Dataset includes DatasetKey, ChangeAuditStamps, VersionTag {
    * List of global tags applied to the dataset
    */
   globalTags: optional GlobalTags
+
+  /**
+   * Related business terms of the dataset
+   */
+  globalGlossaryTerms: optional GlobalGlossaryTerms
```

Comment:
You can rename `GlobalGlossaryTerms` to `GlossaryTerms`. I appreciate you following the tags model, but this is one place you can diverge from tags implementation. We named Tags GlobalTags to differentiate them from an earlier more local tagging system. Since there is no Glossary system we need to differentiate from `GlossaryTerms` works fine.

Reply:
Yeah that can be done. Would need one suggestion:
Tag's model looks like 
`"globalTags": { "tags": [{ "tag": "urn:li:tag:Legacy" }] }`

For glossaryTerm should we have 
`"glossaryTerms": {
       "termUrns": [{
           "urn": "urn:li:glossaryTerm:instruments.FinancialInstrument_v1"
       }]
}`

or 

`"glossaryTerms": {
       "termUrns": ["urn:li:glossaryTerm:instruments.FinancialInstrument_v1"]
}`

Reply:
I would choose `"glossaryTerms": { "terms": [{ "urn": "urn:li:glossaryTerm:instruments.FinancialInstrument_v1" }`

Above is option 1 but I changed `termUrns` to `terms`. I would opt for the first option is that it gives us flexibility to add additional context to a term association later down the line. For example: if we wanted to add ownership to each term association, a timestamp, RBAC information, etc, that would be the location. For this reason, `termUrns` is not a flexible enough name since it does not account for possible additional data we might want to include down the line.

Reply:
Okay, will be making those changes